ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -447,6 +452,17 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             KSQL_SECURITY_EXTENSION_DEFAULT,
             ConfigDef.Importance.LOW,
             KSQL_SECURITY_EXTENSION_DOC
+        ).define(
+            KSQL_ENABLE_TOPIC_ACCESS_VALIDATOR,
+            Type.STRING,
+            KSQL_ACCESS_VALIDATOR_AUTO,
+            ValidString.in(
+                KSQL_ACCESS_VALIDATOR_ON,
+                KSQL_ACCESS_VALIDATOR_OFF,
+                KSQL_ACCESS_VALIDATOR_AUTO
+            ),
+            ConfigDef.Importance.LOW,
+            ""
```

Comment:
Should we add a doc for this?

Reply:
Agree on the doc +1

Reply:
I want this to be an internal config for now. I thought that the way you do that was with an empty docstring, but you actually have to call defineInternal. But then you can't specify a validator. I'm going to choose to not define this as an internal config, so I'll add a doc string. This only matters once we start using auto-doc-generation. And even then, there isn't really any harm in exposing the config, other than adding noise.